docs/src/css/custom.scss

```diff
@@ -52,3 +52,6 @@
 .token.attr-name {
 	color: var(--ifm-attr-name-color) !important;
 }
+.markdown a {
```

Comment:
What does this selector apply to? Does Docusaurus use this for contents it generates from Markdown? Might be worth a comment.

Reply:
Yep, this applies the text decoration to all anchor links within markdown documents, so the links in all our MDX. I'll add a comment